source/core/slang-file-system.cpp

```diff
@@ -319,11 +319,8 @@ CacheFileSystem::CacheFileSystem(ISlangFileSystem* fileSystem, UniqueIdentityMod
 
 CacheFileSystem::~CacheFileSystem()
 {
-    for (const auto& pair : m_uniqueIdentityMap)
-    {
-        PathInfo* pathInfo = pair.value;
+    for (const auto& [_, pathInfo] : m_uniqueIdentityMap)
```

Comment:
Why do we need to modify every iteration to this syntax? Is `KeyValuePair` no longer  a thing we can use explicitly?

Reply:
Not this syntax specifically, but we would have needed to change `key`/`value` to `fst`/`snd`. This was not only a compatible change, but I think makes the code marginally clearer in several places.